transform/convert_quadkey_and_Vertical_id.go

```diff
@@ -342,6 +348,86 @@ func ConvertExtendedSpatialIDsToQuadkeysAndVerticalIDs(extendedSpatialIDs []stri
 	return extendedSpatialIDToQuadkeyAndVerticalID, nil
 }
 
+func ConvertExtendedSpatialIDsToQuadkeysAndVerticalIDsV2(extendedSpatialIDs []string, outputHZoom int64, outputVZoom int64, altitudeRangeScalar int64, verticalIndexOffset int64) ([]*object.FromExtendedSpatialIDToQuadkeyAndVerticalID, error) {
```

Comment:
Rename ConvertExtendedSpatialIDsToQuadkeysAndVerticalIDsV2 to ConvertExtendedSpatialIDsToQuadkeysAndAltitudekeys.
Define a new type FromExtendedSpatialIDToQuadkeyAndAltitudekeys and use it as the returned value.